dev/com.ibm.ws.security.ready.service/src/com/ibm/ws/security/ready/internal/SecurityReadyServiceImpl.java

```diff
@@ -63,7 +64,8 @@
     private volatile boolean securityReady = false;
 
     private ServiceRegistration<SecurityReadyService> reg;
-    private ComponentContext cc;
+
+    private CountDownLatch securityReadyCDL = new CountDownLatch(1);
```

Comment:
I think this should be declared as volatile; which will reduce timing issues when one of the services is deactivating and the CDL is replaced.